### ISSUE TITLE
Mastery reset on login

### ISSUE THREAD

**Issue by Torpedovegas** [WarEmu/WarBugs#6188]
Only today when I logged into the game from crash/stopped playing/ etc... My mastery points were reset and given back to me to spend. I guess it's a free respec but I like my spec lol.

**Comment by Cimbas** [comment]
This happend to me yesterday several times. The first time it happend when I put on the complete mercenary gear on my slayer and joined a szenario/zoned. The build I was trying was http://waronlinebuilder.org/#career=sla;5;5;13;5:;5:;9:13:;;0:0:0:0: 
after the third or fourth time this happend I went back on a 2H spec because that build apparently didn't want to stick around with me. After respeccing on 2H the problem didn't occur again.

I tried to reproduce the bug this morning but wasn't able to. I will do some more testing after work.
